### DISCUSSION TITLE
Alguien más no le enviaron correo de https://launchxauth.azurewebsites.net/ ?

### DISCUSSION THREAD

**Discussion by maxjhael** [Launch-X-Latam/MisionFrontEnd#16] [Q&A]
https://launchxauth.azurewebsites.net/
Espero se resuelva

**Comment by UncannyChimp** [comment]
Ya se comento que de momento se dejara abierto hasta nuevo aviso, por si las dudas estate activo en cuestión de revisar instagram por si avisa de algo

**Comment by fedeminatta** [comment]
igual te recuerdo por las dudas que ese correo te tiene que llegar en la página de Outlook en el correo de innovación que te dieron, no a tu correo personal

**Reply by migueel17** [reply]
Exactamente colega, lo que pasa que aveses se satura la pagina y debes que estar checando constantemente hasta que te de la opcion de vincular el correo pero animo jaja, SUERTE y ojala todos puedan ingresar.

**Comment by doguedogue** [comment]
Manda correo a [contacto@innovaccion.mx](mailto:contacto@innovaccion.mx)
